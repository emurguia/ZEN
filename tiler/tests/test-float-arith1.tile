// testing that float arithmetic between floats works

init {
	float x;
	tile(3,3);
	x = 3.0 + 7.5 * 2.6;
	fprint(x);
	close();
}