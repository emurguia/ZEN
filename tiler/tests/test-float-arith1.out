22.500000
The game window was successfully launched ...
The game window has been exited. Program exited.
Game over.